src/components/Wallets/AccountCard/AccountCardHeader/AccountCardHeader.css

```diff
@@ -26,22 +26,33 @@
 
 .AccountCardHeader .funds .amount {
   display: flex;
-  justify-content: row;
-  align-items: flex-start;
+  justify-content: flex-start;
   align-items: center;
   font-size: 34px;
   font-weight: 600;
   line-height: 42px;
 }
 
+.AccountCardHeader .funds .amount .mana-logo {
+  width: 24px;
+  height: 28px;
+  background-image: url('../../../../images/mana-red-logo.svg');
+}
+
+.AccountCardHeader .funds .amount .matic-logo {
+  width: 24px;
+  height: 24px;
```

Comment:
@braianj - should the same height be used for both logos?

Reply:
looks like the images have those sizes 

<img width="432" alt="Screenshot 2024-06-05 at 10 30 29 AM" src="https://github.com/decentraland/account/assets/6572776/a73a6d1b-80d5-4c77-8da2-49264ac147b5">
<img width="434" alt="Screenshot 2024-06-05 at 10 30 45 AM" src="https://github.com/decentraland/account/assets/6572776/6756eb2c-ae1a-4806-8c8e-de15df232ff4">